### ISSUE TITLE
[Caffe2] Not handle attribute 'spatial' of ONNX operator 'BatchNormalization'

### ISSUE THREAD

**Issue by YuMing-Chang** [pytorch/pytorch#9375]
## Issue description

If I run a ONNX model contains 'BatchNormalization' operator and it has 'spatial' attribute, caffe2 will give me an error message:
```
terminate called after throwing an instance of 'caffe2::EnforceNotMet'
  what():  [enforce fail at backend.cc:1057] . Don't know how to map unexpected argument spatial (from operator SpatialBN)
```

I found that caffe2 do not handle attribute 'spatial' in Caffe2Backend::CreateBatchNormalization:
```C++
Caffe2Ops Caffe2Backend::CreateBatchNormalization(
    OnnxNode* onnx_node,
    int opset_version) {
  if (opset_version < 6) { 
    auto& attributes = onnx_node->attributes;
    attributes.remove("consumed_inputs");
  }

  if (opset_version >= 7) { 
    auto& attributes = onnx_node->attributes;
    auto* attr = attributes.AddRewrittenAttribute("is_test");
    attr->set_i(1);
  }

  return CommonOnnxNodeToCaffe2Ops(onnx_node, opset_version);
}
```

**Comment by JerryShih** [comment]
Here is the example of this BatchNormalization node.
It comes from the official onnx model zoo.
https://github.com/onnx/models/tree/master/models/image_classification/resnet
https://s3.amazonaws.com/onnx-model-zoo/resnet/resnet18v1/resnet18v1.tar.gz

This node contains the caffe2's unhandled attribute "spatial".
From the definition https://github.com/onnx/onnx/blob/master/docs/Operators.md#batchnormalization ,
we could remove the spatial attribute when it equals to 1.

```
node {
    input: "resnetv15_conv0_fwd"
    input: "resnetv15_batchnorm0_gamma"
    input: "resnetv15_batchnorm0_beta"
    input: "resnetv15_batchnorm0_running_mean"
    input: "resnetv15_batchnorm0_running_var"
    output: "resnetv15_batchnorm0_fwd"
    name: "resnetv15_batchnorm0_fwd"
    op_type: "BatchNormalization"
    attribute {name: "epsilon" f: 1e-05 type: FLOAT }
    attribute { name: "momentum" f: 0.9 type: FLOAT }
    attribute { name: "spatial" i: 1 type: INT }
}
```

**Comment by JerryShih** [comment]
Here is the PR #9492 to handle the spatial attrib.

**Comment by bbap-7fat** [comment]
https://github.com/pytorch/pytorch/pull/9492 is to fix the spatial = 1, how about spatial = 0 case?
